backend/database/wrapper/stats/shared_stats_wrapper.py

```diff
@@ -72,6 +72,12 @@ def create_stats_field_list(field_list, explanation_map):
                               DynamicFieldResult('total boost efficiency'), is_percent=True),
             QueryFieldWrapper(stat_math.get_negative_turnover_per_non_dribble(),
                               DynamicFieldResult('turnover efficiency'), is_percent=True),
+            QueryFieldWrapper(stat_math.get_negative_turnover_per_non_dribble(),
```

Comment:
https://media.giphy.com/media/3oz8xIroX3rtDKV1sY/giphy.gif

Should be `stat_math.get_boost_ratio()`.